### PR TITLE
Date slider fix

### PR DESCRIPTION
**Pull request by sandlerben**
@sanjayss34 let's review this code here before incorporating it anywhere else

### CODE REVIEW

**Comment by sandlerben** [comment]
See comments from #29 [here](https://www.dropbox.com/sh/pgik4wxz94ztotp/AACYx5Q6jc0fI9IRk117_jjYa?dl=0)

**Comment by sanjayss34** [comment]
Thanks @nancywong and @kyle-rosenbluth for the good comments; I'm sorry for the very delayed response!

File: app/assets/scripts/map.js [hunk 1]
```diff
@@ -157,6 +157,11 @@ $(document).ready(function() {
     });
 });
 
+function getNextDate(startDate) {
```
Comment (sandlerben) [inline app/assets/scripts/map.js:160]: Can you explain what this does and why it's needed?

Comment (sanjayss34) [inline app/assets/scripts/map.js:160]: This function returns the next date after the given date. The way it's used for setting the date bounds for incident reports is: I pass the end date that the user specifies with the date slider to this function, get back the date after that end date (i.e. the date corresponding to 12:00 AM on the following day) and then put incident reports on the map iff the date of the incident report is before this new date. For instance, if the user specifies January 26, 2015 as the end date, I check the incident reports to see that they were placed before January 27, 2015 at 12:00 AM. The reason to use the next day is that I don't want to filter out any reports that were registered at some time on the end date (in this example, those reports made on January 26). Do you think I should use a different approach?

Comment (sandlerben) [inline app/assets/scripts/map.js:160]: Nope that's great.

**Pull request merged**